WSEvents.cpp

```diff
@@ -642,6 +653,55 @@ void WSEvents::StreamStatus() {
     obs_output_release(stream_output);
 }
 
+void WSEvents::Heartbeat() {
+    bool streaming_active = obs_frontend_streaming_active();
+    bool recording_active = obs_frontend_recording_active();
+    obs_data_t* data = obs_data_create();
+
+    if (Heartbeat_active) {
+        pulse = !pulse;
+        obs_data_set_bool(data, "pulse", pulse);
```

Comment:
What is the purpose of `pulse`?

Reply:
> If the JSON message is every time the same, than you are not assured if things are alright. The toggling pulse shows that the host is alive.

WebSockets relies on the concept of messages, so even if several identical messages are sent by the server, these are still three messages actually sent by the server itself and hence a proof it's alive. So in the case of obs-websocket, receiving events and response to requests from the server is actually an indication that the server is alive, even if the message is always the same.

> In NOCs (Network Operating Centers) the 24x7 monitoring systems do like to see this Heartbeat pulse. If toggle stops an alarm goes of.

What you can do on client side is implement some sort of timeout alert that triggers if no heartbeat has been sent after a defined number of seconds/milliseconds.

Reply:
- I work in telecom and there we also use Nagios to monitor processes in the higher layers of the solution and sent it to a NOC for 24x7 monitoring. Bottom or middle-ware layer can work okay (like websocket), but does also the higher layer of the applications? We also send CPU %,  memory % and free diskspace, so upper/lower thresholds can be set on client side to monitor if the application is in good shape. 
- For me it is very convenient to have pulse like this. Hope it can stay like this.